src/imgtools/ops/sample_loader.py

```
@@ -0,0 +1,147 @@
+                self.dir_path, 
+                series_uid, 
+                file_names
+            ))
+        
+        return images
+    
+    def _reader_RTSTRUCT(self, series_uid: str, reference_image: Scan) -> Segmentation:
+        """Convert RTSTRUCT series to Segmentation using the reference image."""
+        rtstruct_to_segmentation = StructureSetToSegmentation(self.roi_names, continuous=False)
+        struct = self._reader(series_uid)
+        return rtstruct_to_segmentation(struct, reference_image=reference_image, **kwargs)
```

Ruff (0.8.2)

`kwargs` may be undefined, or defined from star imports

(F405)